docs/cpu-stress.yaml

```
@@ -0,0 +1,126 @@
+asyncapi: 2.2.0
+info:
+  title: M-Agent API for CPU Stress Experiment
```

```suggestion
  title: m-agent API for CPU Stress Experiment
```
to match with other references